src/test/java/org/jenkinsci/plugins/scriptsecurity/sandbox/groovy/SandboxInterceptorTest.java

```diff
@@ -556,6 +556,20 @@ public Object invokeMethod(String name, Object args) {
         assertEvaluate(new GenericWhitelist(), 3, "'foo bar baz'.tokenize('ba').size()");
     }
 
+    @Issue("JENKINS-33023")
+    @Test public void enums() throws Exception {
+        String script = "enum Thing {\n"
+            + "  FIRST(\"The first thing\");\n"
+            + "  String description;\n"
+            + "  public Thing(String description) {\n"
+            + "    this.description = description;\n"
+            + "  }\n"
+            + "}\n"
+            + "Thing.values()[0].description\n";
+        String expected = "The first thing";
+        assertEvaluate(new BlanketWhitelist(), expected, script);
```

Comment:
Best not to use `BlanketWhitelist` in tests like this. Rather you want to use either `GenericWhitelist`, or an explicit `StaticWhitelist`.

Anyway although this PR looks like it will fix the `unclassified staticMethod` error, it does not go far enough. We do not want to force the user to whitelist anything we know is safe. In the case of the script shown above, I would have expected [this code](https://github.com/jenkinsci/script-security-plugin/blob/1c321f31b5b382390d2509a699cd7516f7e25a5c/src/main/java/org/jenkinsci/plugins/scriptsecurity/sandbox/groovy/GroovySandbox.java#L136) to automatically whitelist the initializer.

Perhaps this is just a matter of the test not reflecting what the code change is actually accomplishing.